x/upgrade/alias.go

```diff
@@ -32,7 +32,7 @@ var (
 )
 
 type (
-	UpgradeHandler                = types.UpgradeHandler
+	Handler                       = types.UpgradeHandler
```

Comment:
Why this change?

Reply:
golint throws linter error when how you call a pkg is also the first part of a func (upgrade.UpgradeHandler)

Reply:
Add a lint ignore. We shouldn't be renaming variables/types on aliasing. We need to address this in a follow-up on `master`.